projects/ngrx.io/content/guide/eslint-plugin/rules/prefer-protected-state.md

```diff
@@ -17,28 +17,32 @@ This rule ensures that state changes are only managed by the Signal Store to pre
 
 Examples of **incorrect** code for this rule:
 
+<!-- prettier-ignore -->
 ```ts
-// SUGGESTION ❗
 const Store = signalStore(
   { protectedState: false },
-    ~~~~~~~~~~~~~~~~~~~~~ [warning]
+//  ~~~~~~~~~~~~~~~~~~~~~ [ESLint: { protectedState: false } should be removed to prevent external state mutations.]
   withState({}),
 );
 ```
 
 Examples of **correct** code for this rule:
 
+<!-- prettier-ignore -->
 ```ts
-// GOOD ✅
 const Store = signalStore(
   withState({}),
 );
 ```
 
+<!-- prettier-ignore -->
 ```ts
-// GOOD ✅
 const Store = signalStore(
   { protectedState: true },
   withState({}),
 );
 ```
+
+## Further reading
+
+- [SignalStore's state protection](guide/signals/signal-store#signalstores-state-protection)
```

Comment:
Keep in mind that when we remove the "### SignalStore's state protection" header, that we also need to update this link (or remove it).